src/views/Home/index.tsx

```
@@ -25,19 +27,36 @@ const Home = () => {
-    </>
+          {primaryRole && (
+            <span className='mt-3 inline-flex rounded-full border border-accent-100 bg-accent-50/80 px-3 py-1 text-xs font-semibold text-accent-700'>
+              Rol: {primaryRole}
+            </span>
+          )}
+        </div>
+        <button
+          onClick={logout}
+          title='Salir de la aplicación'
+          className='group inline-flex items-center gap-2 self-start rounded-lg border border-primary-200 bg-white/85 px-3 py-2 text-sm text-primary-600 transition-colors hover:border-error-200 hover:bg-error-50 hover:text-error-600'
+        >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)